src/components/SettingsModal.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useContext, useEffect } from "react";
+import reactDom from "react-dom";
+import classnames from "classnames";
+import { v4 as uuidv4 } from "uuid";
+import { ModalContext } from "./ClockFace.js"
+import "./SettingsModal.scss";
+
+const SettingsModal = () => {
+  const { isModalOpen, setIsModalOpen, sessions, setSessions } = useContext(ModalContext);
+  const [localSession, setLocalSession] = useState(sessions);
```

Comment:
this is for holding the user's temporary settings on the modal, since the user can click "Close" and not save, and the app still has the previous settings